activemodel/test/cases/validations/acceptance_validation_test.rb

```diff
@@ -8,7 +8,7 @@
 
 class AcceptanceValidationTest < ActiveModel::TestCase
   teardown do
-    self.class.send(:remove_const, :TestClass)
+    self.class.send(:remove_const, :TestClass) if self.class.const_defined?(:TestClass)
```

Comment:
Is this change necessary?

Reply:
Ah sorry no, it was before but I force-pushed a change so it's no longer  required. I'll remove, thanks.